### PR TITLE
make choose camera in setting become effective

### PR DESCRIPTION
**Pull request by WenyuanWu99**
change the order about full_renderer and small_renderer to make small_renderer show normally

fix bug #115 use System.exit(0) to release resource in onDestory()
fix bug #102 the way to unpublish in P2P will lead crush after serveral times publish and unpublish, so change it to another way
~~fix bug #94  use cyclicBarrier to avoid use info() when conferenceInfo is null~~

### CODE REVIEW

File: src/sample/conference/src/main/res/layout/fragment_video.xml [hunk 1]
```diff
@@ -39,16 +39,18 @@
...
         <org.webrtc.SurfaceViewRenderer
             android:id="@+id/full_renderer"
             android:layout_width="match_parent"
             android:layout_height="match_parent"/>
 
+        <org.webrtc.SurfaceViewRenderer
+            android:id="@+id/small_renderer"
+            android:layout_width="160dp"
+            android:layout_height="120dp"/>
+
```
Comment (jianjunz) [inline src/sample/conference/src/main/res/layout/fragment_video.xml]: Please don't submit changes that are unnecessary.

File: src/sdk/conference/src/main/java/owt/conference/ConferenceClient.java [hunk 2]
```diff
@@ -21,9 +21,13 @@
 import java.util.Collections;
```
Comment (jianjunz) [inline src/sdk/conference/src/main/java/owt/conference/ConferenceClient.java]: #94 has been fixed by #123. My idea to make everything ready before firing success callback.

File: src/sample/conference/src/main/res/layout/fragment_video.xml (hunk 1 above)
Comment (WenyuanWu99) [reply src/sample/conference/src/main/res/layout/fragment_video.xml]: Sometimes the full_renderer will cover the small_renderer if it above small_renderer.So I put it down.
If the small_renderer is useless, I will put it back.

File: src/sdk/conference/src/main/java/owt/conference/ConferenceClient.java (hunk 2 above)
Comment (WenyuanWu99) [reply src/sdk/conference/src/main/java/owt/conference/ConferenceClient.java]: Ok, I remove these change.

**Pull request merged**